### ISSUE TITLE
Add jappanese lrnguage

### ISSUE THREAD

**Issue by canochento** [klausw/hackerskeyboard#775]
Is possible?

**Comment by Vortetty** [comment]
yes adding a japanese keyboard would be possible, although it would probably be very hard due to 3 alphabets larger than the english one this is based on, and having to make the code to determine what alphabet to use for each word would be hard to say the least.

**Comment by Vortetty** [comment]
@canochento i will look at adding this for you (and anybody else using japanese) i will look at chinese however it wont work very well, default would be hiragana, shift/caps lock would get you katakana, and the function menu would give you kanji

**Comment by canochento** [comment]
thanks.
I love your app.
